lib/plist/parser.rb

```diff
@@ -73,10 +71,10 @@ def initialize( plist_data_or_file, listener )
     end
 
     TEXT       = /([^<]+)/
-    XMLDECL_PATTERN = /<\?xml\s+(.*?)\?>*/um
-    DOCTYPE_PATTERN = /\s*<!DOCTYPE\s+(.*?)(\[|>)/um
-    COMMENT_START = /\A<!--/u
-    COMMENT_END = /.*?-->/um
+    XMLDECL_PATTERN = /<\?xml\s+(.*?)\?>*/m
```

Comment:
Can you explain why you decided to remove the `u` option on these patterns? I am confused as to why they were there in the first place. Can you speculate as to their original purpose?

I agree with not using `u`, but I am worried we might break something by removing them. Is it a Ruby 1.8.7 thing?